paddle/fluid/operators/mkldnn/layer_norm_mkldnn_op.cc

```diff
@@ -150,4 +150,5 @@ class LayerNormMKLDNNOpKernel : public paddle::framework::OpKernel<T> {
 // TODO(jczaja): Enable FP32 when performance is good
 namespace ops = paddle::operators;
 REGISTER_OP_KERNEL(layer_norm, MKLDNN, ::paddle::platform::CPUPlace,
+                   ops::LayerNormMKLDNNOpKernel<float>,
```

Comment:
Hi, so this PR enabled oneDNN layernorm? Then the line `//TODO...` could be removed? It maybe removed in later PRs together with other functions?
```
// TODO(jczaja): Enable FP32 when performance is good
namespace ops = paddle::operators;
REGISTER_OP_KERNEL(layer_norm, MKLDNN, ::paddle::platform::CPUPlace,
                   ops::LayerNormMKLDNNOpKernel<float>,
```

Reply:
@lidanqing-intel  Yes. This comment could be removed.